README.md

```diff
@@ -9,59 +9,35 @@ This branch is the development branch for the OpenStudio workflow gem.
 
 The OpenStudio Workflow Gem has the following dependencies:
 
-* Ruby 2.0
-* OpenStudio with Ruby 2.0 bindings
-* EnergyPlus 8.3 (assuming OpenStudio >= 1.7.2)
-* MongoDB if using MongoDB Adapter (or when running rspec)
+* Ruby 2.2.4
+* OpenStudio 2.x
```

Comment:
@nllong are you planing a release after this one to bump to ruby 2.5.5 / OpenStudio 3.x ? (cf https://github.com/NREL/OpenStudio-workflow-gem/pull/98)